charts/kyverno/values.yaml

```diff
@@ -261,14 +261,15 @@ config:
   # Note that it takes a list of `namespaceSelector` and/or `objectSelector` in the JSON format, and only the first element
   # will be forwarded to the webhook configurations.
   webhooks:
-    # Exlude namespaces
+    # Exclude namespaces
     # - namespaceSelector:
     #     matchExpressions:
-    #     - key: environment
+    #     - key: kubernetes.io/metadata.name
     #       operator: In
     #       values:
-    #         - prod
-    # Exlude objects
+    #         - kube-system
+            # - kyverno
```

Comment:
Is kyverno namespace hard-coded here? What if Kyverno is installed in a different namespace?

Reply:
This is just a template which is commented out by default. Users should change the value as needed to match the destination Namespace.